### DISCUSSION TITLE
More details on nhost deploy error?

### DISCUSSION THREAD

**Discussion by caudurodev** [nhost/nhost#2470] [Q&A]
Currently my project fails to build in production and has the error message in the dashboard:

`Error deploying the project most likely due to invalid configuration. Please review your project's configuration and logs for more information.`

This is the deploy log:

```
08:22:51:
Starting deployment
08:22:51:
Cloning repo github.com/[examplereponame]
08:22:53:
Detected new config file
08:27:57:
Deployment completed with status FAILED
```

The same code deploys fine in the CLI (localhost) and the only changes made were to hasura actions in the new code which fails. After resetting the changes I still incur the same errors. How can I get better insight into what is failing in the nhost live hosting?

I'm sure I broke something, but struggling to get a hint as to what. If the deploy error was a little more verbose it would help to debug...

**Comment by dbarrosop** [comment]
Application logs should be able to give you more information. Otherwise, feel free to contact us on discord or via email and provide the subdomain so we can take a look.

**Reply by caudurodev** [reply]
Where are the application logs? I can see logs for the deployed services in `https://app.nhost.io/user/application/logs`, but will my failed deploy have logs there too?

**Reply by dbarrosop** [reply]
That's the correct URL and, yes, logs indicating why the deploy is failing may show up in there as the issue is probably that a service is failing to apply the new configuration.

**Reply by caudurodev** [reply]
I see no additional logs on that URL on failed deploys... maybe I need to open a ticket.

**Reply by dbarrosop** [reply]
For the record, this was resolved on discord. Thanks for reaching out!
